pkg/controller/jobframework/reconciler.go

```diff
@@ -437,6 +437,7 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 					workload.SetRequeuedCondition(wl, evCond.Reason, evCond.Message, true)
 				}
 				_ = workload.UnsetQuotaReservationWithCondition(wl, "Pending", evCond.Message)
+				r.record.Event(wl, corev1.EventTypeNormal, kueue.WorkloadEvicted, evCond.Message)
```

Comment:
I one-liner fixes :). 

It would be great to cover it with some unit or integration tests. IIRC we already have some unit tests for events, maybe we can employ the same way.

Reply:
Thanks